{"name": "test_one", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "self = <camp_study_interface.aek.testcase.test_login.TestLogin object at 0x0000017088EC4050>\n\n    def test_one(self):\n        print(\"hello\")\n        q=1\n        w=2\n>       assert  q==w\nE       assert 1 == 2\n\ntest_login.py:28: AssertionError"}, "start": 1676208872700, "stop": 1676208872700, "uuid": "f052ec05-d384-4f23-bace-2bdddb11ea08", "historyId": "e8a06419f3944897a26c18284a1c4693", "testCaseId": "76b312f8ab2a9e29d1b57afd4bf7f6f5", "fullName": "test_login.TestLogin#test_one", "labels": [{"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "我叫小白菜"}, {"name": "thread", "value": "14908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}